packages/playground-examples/copy/ja/TypeScript/Primitives/Literals.ts

```diff
@@ -0,0 +1,65 @@
+// TypeScriptには、ソースコードにリテラルを用いた
+// 特別な面白いケースがあります。
+
+// これは、型の拡張・型の絞り込みにおいて、多くのサポートをもたらします。
+// （例：type-widening-narrowing）
```

Comment:
You probably want to keep this as `example: type-widening-narrowing ` because it becomes clickable inside the playground to take you to that example